wrangler/src/txFunctions/run.js

```diff
@@ -149,8 +77,8 @@ export default async ({ request, params, env, ctx }) => {
     xdr,
     error,
     cost,
-    feeTotal,
-    feeSpent
+    feeSponsor,
```

Comment:
Is it possible to use the fee sponsorship form another account? If a wallet could maintain fee balances with all public TSS nodes and then just sponsor its users function calls, this would drastically reduce the overall number of fee settlement transactions. I think major wallets will be happy to either allow their users to run txFunctions free of charge (of course, with throttling in order to prevent spam attacks) or just take care of TSS fees settlement by charging a single payment from an account (vs managing a few dozens fee balances).

Reply:
Apologize if that was specifically the case here. I lost track in the discussions.

Reply:
Orbit and I talked a bit in KB about this. Quick summary below to keep discussion in same place:

Currently, each account can only use their own Fee Balance. A Service / Sponsor can pay a fee payment on behalf of a user, such that the fee balance is tracked by the USER on the Turret, not the Service / Sponsor.

This has two advantages:
1. Wide scaling KV stores for individual fee balances is a common problem, and can be easily solved in the cloud
2. A user can pay fees to a turret, and use them on any web app, service, or wallet.

Eventually, if Stellar Turrets becomes quite popular, we will want to support the ability to add some sort of fee sponsorship mechanism for large wallets like Lobstr can easily integrate.

The thought to do this is that a Service Account would allow (mechanism TBD, likely in token) a user to use the service / sponsor accounts fee balance when running against the turret.

This makes integrating much easier, but comes with two things to watch out for.
1. Turrets must support concurrent fee requests. Some micro service should exists that can order and process fee payments and accruals.
2. It might be a bad user experience if they have to pay fees to a wallet provider that controls a sponsor account, a web app that supports a sponsor account, and other services, all to access the same turret.

Reply:
So am I right in assume tldr; no changes needed for this PR?

Reply:
Yes, no changes required unless we want to add some basic scaffolding for this feature in advance.